VastClient/VastClient/models/XML schema/VastLinearCreative.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  VastLinearCreative.swift
+//  VastClient
+//
+//  Created by John Gainfort Jr on 4/6/18.
+//  Copyright © 2018 John Gainfort Jr. All rights reserved.
+//
+
+import Foundation
+
+struct CreativeLinearElements {
+    static let duration = "Duration"
+    static let adParameters = "AdParameters"
+
+    // /MediaFiles
+    static let mediafile = "MediaFile"
+    static let interactiveCreativeFile = "InteractiveCreativeFile"
+    
+    // /VideoClicks
+    static let clickthrough = "ClickThrough"
+    static let clicktracking = "ClickTracking"
+    static let customclick = "CustomClick"
+    
+    static let trackingEvents = "TrackingEvents"
+    static let tracking = "Tracking"
+    
+    // /Icons
+    static let icon = "Icon"
+}
+
+fileprivate enum LinearCreativeAttribute: String, CaseIterable {
+    case skipoffset
+    
+    init?(rawValue: String) {
+        guard let value = LinearCreativeAttribute.allCases.first(where: { $0.rawValue.lowercased() == rawValue.lowercased() }) else {
+            return nil
+        }
+        self = value
+    }
+}
+
+// VAST/Ad/InLine/Creatives/Creative
+// VAST/Ad/Wrapper/Creatives/Creative
+public struct VastLinearCreative {
+    public let skipOffset: String? // TODO: Consider changing to Int
+    
+    public var duration: Double? // Inline only
+    public var adParameters: VastAdParameters? // Inline only
+    public var videoClicks: [VastVideoClick] = []
+    public var trackingEvents: [VastTrackingEvent] = []
+    public var mediaFiles = VastMediaFiles()
```

Comment:
Accessing a url requires something like

`linearCreative.mediaFiles.mediaFiles.first(where: { ... })`

Im guessing there was a strong reason to have a VastMediaFiles struct so maybe you could change its property to files or something?

Reply:
This comes from the VAST 4.0 XML schema, where they named a structure `MediaFiles` which contains `Mezzanine` `MediaFile` and `InteractiveCreativeFile`. 

https://www.iab.com/wp-content/uploads/2016/04/VAST4.0_Updated_April_2016.pdf page 43.

I created the structure with elements name to copy the style of documentation. 

We can remove the whole container structure for MediaFiles or rename it. Which one would you preffer @jgainfort ?

Reply:
I think rename is easiest